Integrations/integration-PagerDuty.yml

```diff
@@ -246,11 +260,11 @@ script:
             'ReadableContentsFormat': formats['markdown'],
             'HumanReadable': tableToMarkdown(SCHEDULES, outputs, SCHEDULES_HEADERS),
             'EntryContext': {
-                'PagerDuty.Schedules': context
+                'PagerDuty.Schedules': contexts
```

Comment:
@ronykoz should be `PagerDuty.Schedules(val.id==obj.id)`

Reply:
Same as above